main.js

```diff
@@ -0,0 +1,46 @@
+class Game {
+board = [[5, 4, 3, 2, 1],[],[]];
+
+//using map  to create a starting board mathod within the game class/object
+ startBoard() {this.board = this.board.map(function(towers){
+    console.log(`--- ${towers.join('')}`);
+    return towers
+ });
+}
+
+//method that allows movement of discs between towers, with parameters regarding legal moves using 'if/else'
+moveDisc (fromTower, toTower){
+      if (this.board[fromTower].length === 0){
+        console.log("There is no disc on this tower to move. Pick a different move please");
+        return;
+      }
+
+      if (this.board[toTower].length === 0 || this.board[fromTower][this.board[fromTower].length - 1] < board[toTower][board[toTower].length - 1]) {
+        var disc = this.board[fromTower].pop();
+        this.board[toTower].push(disc);
+        console.log(`Moved disc ${disc} from tower ${fromTower + 1} to tower ${toTower + 1}`);
```

Comment:
string interpolation is an es6 feature, it would be good to stay consistent in the code (ie use const/let when declaring variables